src/Model/RequestManager.php

```diff
@@ -19,10 +19,18 @@ public function __construct()
         parent::__construct(self::TABLE);
     }
 
-    public function selectFirsts(): array
+    public function selectFirsts(): ?array
     {
-        return $this->pdo->query('SELECT * FROM ' . self::TABLE . ' JOIN ' . UserManager::TABLE .
-                            ' ON user.id = fk_requester_id LIMIT 6')->fetchAll();
+        try {
+            $statement = $this->pdo->query('SELECT * FROM ' . self::TABLE . ' JOIN ' . UserManager::TABLE .
```

Comment:
You statement variable contains the data (because of the fetch) not the statement